driver/base/options.go

```diff
@@ -218,6 +218,21 @@ func WithNetconfServerEcho(echo bool) Option {
 	}
 }
 
+// WithNetconfForceSelfClosingTags forces empty tags to the form </tag> -- only applicable for
+// netconf, and *probably* only for junos.
+func WithNetconfForceSelfClosingTags(echo bool) Option {
```

Comment:
Hey mate
Why the arg name is echo?

Reply:
blah, good catch. copy pasta lazy Sunday I guess 🤷